lib/protoboeuf/codegen.rb

```diff
@@ -89,8 +247,13 @@ def result(message, toplevel_enums, generate_types:, requires:, syntax:, options
         end
       end
 
-      attr_reader :message, :fields, :oneof_fields, :syntax
-      attr_reader :optional_fields, :enum_field_types
+      attr_reader :message,
+        :fields,
+        :oneof_fields,
+        :syntax,
+        :optional_fields,
+        :enum_field_types,
+        :oneof_selection_fields
```

Comment:
As long as you're updating these lines, what do you think about moving `:generate_types, :requires` (line 240) into it just to have them all together?  I found myself wondering where the attr's below were defined until I expanded the the diff to see that the ones above are in the same class.